docs/docs/development/setup/getting-started.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Getting Started
+sidebar_label: Getting Started
+---
+
+:::info
+Throughout this documentation you will see commands like
+
+```sh
+sudo apt update
+```
+
+These commands should be run in a terminal such as Bash (Linux/MacOS/Docker), PowerShell, or Command Prompt (Windows).
+:::
```

Comment:
Do you think this is necessary? I feel like this level of detail doesn't belong in dev documentation. I'd probably not recommend the user to use the local build option if this is a point of confusion.

Reply:
It depends on the target audience. Anyone with dev experience shouldn't need this note, but I think it may be helpful for some less experienced windows users. It does contribute to making it too verbose though, so I'll get rid of it.